server/channels/api4/limits.go

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+package api4
+
+import (
+	"encoding/json"
+	"net/http"
+
+	"github.com/mattermost/mattermost/server/public/model"
+
+	"github.com/mattermost/mattermost/server/public/shared/mlog"
+)
+
+func (api *API) InitLimits() {
+	api.BaseRoutes.Limits.Handle("/users", api.APISessionRequired(getUserLimits)).Methods("GET")
+}
+
+func getUserLimits(c *Context, w http.ResponseWriter, r *http.Request) {
```

Comment:
0/5 do we want to add tests at api level?

Reply:
I think its fine as there is no logic in API layer other than checking for admin role. We can add later as logic grows.